packages/renderer/src/lib/preferences/PreferencesExtensionList.svelte

```diff
@@ -83,7 +83,10 @@ async function updateExtension(extension: ExtensionInfo, ociUri: string) {
       <h1 class="text-lg mb-2">Install a new extension from OCI Image</h1>
 
       <div class="flex flex-col w-full">
-        <div class="flex flex-row mb-2 w-full space-x-8 items-center">
+        <div
+          class="flex flex-row mb-2 w-full space-x-8 items-center"
+          role="region"
+          aria-label="InstallExtensionFromOCI">
```

Comment:
Use spaces between words, everywhere in aria-label.

Reply:
fixed